scripts/download_wheels.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+#
+# Download all wheels for a pandas version.
+#
+# This script is mostly useful during the release process, when wheels
+# generated by the MacPython repo need to be downloaded locally to then
+# be uploaded to the PyPI.
+#
+# There is no API to access the wheel files, so the script downloads the
+# website, extracts the file urls from the html, and then downloads it
+# one by one to the dist/ directory where they would be generated.
+
+VERSION=$1
+DIST_DIR="$(realpath $(dirname -- $0)/../dist)"
+
+if [ -z $VERSION ]; then
+    printf "Usage:\n\t$0 <version>\n\nWhere <version> is for example 1.5.3"
+    exit 1
+fi
+
+curl "https://anaconda.org/multibuild-wheels-staging/pandas/files?version=${VERSION}" | \
+    grep "href=\"/multibuild-wheels-staging/pandas/${VERSION}" | \
+    sed -r 's/.*<a href="([^"]+\.whl)">.*/\1/g' | \
+    awk '{print "https://anaconda.org" $0 }' | \
+    xargs wget -P $DIST_DIR
+
+printf "\nWheels downloaded to $DIST_DIR\nYou can upload them to PyPI using:\n\n"
```

Comment:
Maybe a small QOL improvement would be to print which wheels were downloaded?

Reply:
Not sure if I know how to do it without making the script significantly more complex. I can just do a `ls dist/` but assuming there were wheels in that directory it can report wrong information.

It's not super nice, but the log of wget shows the file names as they are downloaded. There is some noise in between files, but my preference would be to just have that. The alternatives I see are to save the url list in a temporary file, and then print the file content with a regex to remove the parts of the url that are not the file. Not worth in my opinion.